libs/wingsdk/src/std/resource.ts

```diff
@@ -168,12 +167,6 @@ export abstract class Resource extends Construct implements IResource {
    * @internal
    */
   public _registerBind(host: IInflightHost, ops: string[]) {
-    log(
```

Comment:
The only log in the village, so remove for now...

Reply:
I'd be lying if I said I haven't used this at least once or twice for debugging the binding system with Hasan and others. 🙂 How about we keep it and do this instead?

```js
if (process.env.DEBUG) {
  console.error(<message>);
}
```